### PR TITLE
International pricing

### PR DESCRIPTION
**Pull request by natalieoldroyd**
International Pricing

### CODE REVIEW

**Review by syfiawoo — COMMENTED** [review]
Looking good! A comment I had was to incorporate the use of variables rather than hardcoding them within the queries. The [customers sec](https://github.com/Shopify/storefront-api-learning-kit/tree/main/examples/07_customers/01_create_an_access_token) can be used as a template for creating query variables.

File: examples/03_international_pricing/05_create_cart_in_local_context/query.graphql [hunk 1]
```diff
@@ -0,0 +1,43 @@
+#If a store has setup additional Markets and has translated resources, a localized cart can be generated.
+#The link to the cart below will be presented in the default currency of the country passed in and in the language passed in if a matching translated resource is available.
+#If no language is passed in the localized cart will be presented in the default language of country context and in that market's currency setup in Admin.
+
+mutation cartCreate($cartInput: CartInput!)@inContext(country:US, language:ES) {
```
Comment (syfiawoo) [inline examples/03_international_pricing/05_create_cart_in_local_context/query.graphql:5]:
```suggestion
mutation createCart($cartInput: CartInput!, $country: CountryCode, $language: LanguageCode)@inContext(country: $country, language: $language) {
```

File: examples/03_international_pricing/05_create_cart_in_local_context/query.graphql [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+	cartCreate(input: $cartInput) {
```
[the remark this answers is not in the document]
Comment (natalieoldroyd) [reply examples/03_international_pricing/05_create_cart_in_local_context/query.graphql:6]: Done

**Review by syfiawoo — COMMENTED** [review]
All looks good. Just a few nits.

Also I see we also have a `getCustomerOrders` query under customers and it got me thinking if we should have similar queries under different topics. Something we can discuss at our next sync

File: examples/03_international_pricing/02_get_product_prices/query.graphql [hunk 3]
```diff
@@ -1,11 +1,11 @@
...
   products(first: 1) {
     edges {
       node {
         variants(first:1) {
           edges {
             node {
-              priceV2 {
+              price {
                 amount
                 currencyCode
```
Comment (syfiawoo) [inline examples/03_international_pricing/02_get_product_prices/query.graphql:10]: Could you add some additional fields to identify product and variant information. Maybe just titles

File: examples/03_international_pricing/04_get_customer_orders/query.graphql [hunk 4]
```diff
@@ -1,23 +1,23 @@
...
                   amount
-                  currencyCode # order's currency - USD (point in time)
+                  currencyCode # store's currency - CAD in this case
                 }
                 variant {
-                  priceV2 {
+                  price {
                     amount
-                    currencyCode # EUR variant's currency (past context)
+                    currencyCode # EUR currency (if French market is enabled for store)
```
Comment (syfiawoo) [inline examples/03_international_pricing/04_get_customer_orders/query.graphql:20]: Same comment as above. Describing the field without calling out currencies. For example, localized currency

**Review by syfiawoo — APPROVED** [review]
🚀

**Pull request merged**